simple-backend/stave_backend/lib/stave_viewer.py

```diff
@@ -12,17 +12,17 @@
 
 import os
 import json
-import errno
 import logging
 import asyncio
 import threading
 import webbrowser
-from typing import Dict, Set, Any, List
+from typing import Optional
 
 import django
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.core.management import call_command
+from django.core.management.utils import get_random_secret_key
```

Comment:
Looks like we use some django api to create these. Is it possible that if we use `django-admin startproject` directly (when the user didn't provide a django project path)?